apps/etterlatte-tilbakekreving/src/main/resources/db/migration/V2__legg_til_sakid.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE tilbakekreving_hendelse ADD COLUMN sak_id BIGINT;
+ALTER TABLE tilbakekreving_hendelse ALTER COLUMN kravgrunnlag_id DROP NOT NULL;
```

Comment:
Bør vi også ha eit migreringsskript som definerer sak_id for dei eksisterande innslaga, eller kan vi sjå på dei som historiske/behandla og la dei vera som dei er?

Reply:
Jepp, det står på planen. Lager jira-oppgave for det

Reply:
https://jira.adeo.no/browse/EY-4047